### PR TITLE
Alan phase1 branch

### PR DESCRIPTION
**Pull request by alanchen34**

### CODE REVIEW

**Review by Everett02 — CHANGES_REQUESTED** [review]
We need to separate 'add' method and 'search' method.

File: src/main/java/Controller/PickupSystem.java [hunk 1]
```diff
@@ -6,31 +6,70 @@
...
-    public void storeItem(){
+    public boolean storeItem(String id, List<String> info, String storageRequirement){
         // this will interact with the UI layer
+        iman.createItem(String id, List<String> info, String storageRequirement);
+        if(iman.addItem(id,currentUser)==false){
+            return false;
+        }
+        return true;
+
+    public boolean search() {
```
Comment (QI777Q) [inline src/main/java/Controller/PickupSystem.java:57]: Fulfill this method and add a remove method

Comment (alanchen34) [reply src/main/java/Controller/PickupSystem.java:57]: ok got it

File: src/main/java/UseCase/UserManager.java [hunk 2]
```diff
@@ -16,10 +16,13 @@ public boolean register(){
         return true;
     }
 
-    public boolean lookupUser(String username, String pw){
-        // look up the user in umap; if not found, return false
-        return true;
-    }
```
Comment (Everett02) [inline src/main/java/UseCase/UserManager.java:22]: This is a repetitive method. Delete it to resolve conflict.

**Pull request closed without merging**